src/main/java/gyro/google/compute/NetworkEndpointResource.java

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright 2019, Perfect Sense, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package gyro.google.compute;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import com.google.api.services.compute.model.HealthStatusForNetworkEndpoint;
+import com.google.api.services.compute.model.NetworkEndpoint;
+import com.google.api.services.compute.model.NetworkEndpointWithHealthStatus;
+import gyro.core.resource.Diffable;
+import gyro.core.resource.Id;
+import gyro.core.resource.Output;
+import gyro.core.validation.Required;
+import gyro.google.Copyable;
+
+public class NetworkEndpointResource extends Diffable implements Copyable<NetworkEndpointWithHealthStatus> {
+
+    private InstanceResource instance;
+    private Integer port;
+    private String ipAddress;
+
+    // Read-only
+    private List<String> healthStatus;
+
+    @Id
```

Comment:
No @Id annotation for a subresource, as findById only works for resources
```suggestion
```

Reply:
Done.